TD Balance 20180520

AoAGeneral opened the pull request:
Apache damage vs infantry at prone increased from 50 percent to 80 percent.

  Apache damage was cut in half making them less effective versus infantry. Giving them an extra boost damage wise versus prone is enough then going with a universal damage boost.

MCV Price from 4000 to 3500.

  Some say they want 3000 but im still paranoid of the epidemics that have happened in the past with 1v1 and team games. Needs more team game tests if they want 3000.

Repair Pad decrease power from -30 to -20.

 Power consumption is a little high. Specifically if they want to build extras.

APC Price from 550 to 600.

APC HP from 21500 to 19000.

APC turn speed from 8 to 5.

APC build duration from 900 to 938.

APC AA range from 7 to 5.

APC reduce vision range from 7c0 to 6c0.

APC projectile speed reduced from 2c0 to 900.

  See notes below.

Power Plant buff HP from 50,000 to 55,000

  Makes power plant sniping a little tougher. But still effective to take out.

Change power structures (Airstrip/Factory and Refinery to 40 power)

  Important change. This allows more diverse builds to happen such as now able to build refinery > barracks > factory. This combo helps easy defends with quick vehicle herasses.

MRLS Price reduction from 1000 to 900.

  Allows an easier method to build on the field. A unit that is often times killed by airstrikes allows more forgiveness when on the field.

Light vehicle husks HP reduce to 2000. (reduce timer on field wreckage.)

  Prevents early on refinery blocks and other pathing unit issues.

Oil Derrick reduce HP from 100,000 to 80,000

  Oil Derricks were a little to strong. Easily can be killed now.

Change power plant build duration from 12 seconds to 8 seconds. (Low power build duration from 24 seconds to 16 seconds.) Note: Does not include Adv. Power plant

  Due to the struggle of low power this enables players to build regular power plants to help get back up quickly. This however, does not apply to Adv. power plants. The outcome is then scattered extra power plants rather then Adv. power due to the player being on low power.

Change engineer capture threshold from 50 to 55.

  Fixes the structures from being fixed and preventing the second engineer from getting canceled out or being wasted on damaging the structure again.

NOTES:

I have mentioned that I disagree on the above APC changes. The reasons I conclude may happen is buffing the bikes with a lack of counter. This may result in games where infantry are now used to help counter against buggy/bike play rather then an APC/infantry play or APC/hummer play. If and when players speak about bikes being OP please look back at the changes that happened in the past.

ZxGanon commented:
I dissaprove with the changes since it hasnt been tested with enough people. Also the changes are not enough to help out TD coming out of it´s depression.

abcdefg30 commented:
Going from what you said here and on IRC you are fine with the changes made, just want some additional polish before the next release. We could merge this PR to create a common base to work the next set of balance PRs for TD out.

ZxGanon commented:
Okay now I got the time to answer.

To confirm what @abcdefg30 said: Yes the changes @AoAGeneral posted above are definitely an improvement for OpenRA Tiberian Dawn. My quick and aggressive response was caused by the fear that again changes get pushed through without any polishing or consideration of the community, even though that this game doesnt have the playerstrength of OpenRA Red Alert 1.

There were several events in the past that led me to the conclusion that it is gonna happen again.

Anyway my two concerns for a new PR and with that a possible shake up of the current TD meta and gameplay overall are as follow:
1. A rushed PR that just only includes the changes mentioned above from AoAGeneral and only these will be taken over.
2. The community didnt had any chance to atleast maybe leave a comment about it.

I have been developing mod maps together with Mesacer, Anjew and other community members to test possible improvements for TD that AoA didnt think of.

For example:
* Decreasing the HP of Oil Derricks further from 100000 to 70000
* Aswell as further decreasing the cost of Rocket Launchers from 1000$ to 800$
* Rocket Solider: Increased movment speed from 42 to 50

Those kind of changes and other would fall under the category of predictable to not break the game.

While other changes are a bit bold and try to go further to shake up the current meta:
MCV:
* Cost is reduced to 3000$
* Build time changed from 60 to 47 seconds
* HP reduced from 120000 to 60000
* Speed reduced to 60 

An other changes that either add a new gameplay aspect like:
New starting units:
* Scout Squad which provides you with 3 Minigunner
This has been a request since the current Light Support weakens Nod´s early game by a lot and not everyone prefers to play with MCV only. It provides you with scouting units but also provides a fair atmosphere for both factions.
Airstrike:
* Changed so that it only appears from either north or south, still random appearance
We only have been testing this for 3 days but so far it has improved the gameplay as exspected. Both players knew there was the possibility of an Aistrike but the direction was known. The only problem is that its still randomized, means if you are next to the north or south border of the map the airstrike can either come from the closest or the most far away position. But airstrikes are with that change less RNG and more predictable. Of course there are maps like Pirates and Emperors where both players spawn on the North and South position and airstrikes can immediately effect each others base but it is certainly more calculated than a random AS coming from the side and obliterating your whole army like it was seen in the first match of TDGL between Orb and me where my AS cam from the closest position with the perfect angle to kill 7 Attack Bikes and render his attack useless.

There are also changes that were planned for the next mod update. Pchote once said that he wanted to change the experience gain system and I thought about implementing the current system we have in Shattered Paradise since a month:
Changed veterancy system:
Veterancy: (achieved after killing 300% of the units worth)
+10% Durability
+10% Attack damage
+10% Attack speed
+10% Movement speed
Elite: (achieved after killing 600% of the units worth)
+10% Durability
+10% Attack damage
+10% Attack speed
+10% Movement speed
Heroic: (achieved after killing 900% of the units worth)
Unit gains self healing to 100%
+10% Durability
+10% Attack damage
+10% Attack speed
+10% Movement speed
Removed Chevron 3 and made Platinum Star the new Rank 3
Removed accuracy bonus
Streamlined the other bonuses
Adjusted required experience to gain rank
So far it has been proven very well. Of course it is still the case that infantry reaches heroic faster than a Mammoth Tank but the new system first improved the veterancy among units and the bonus it provides is also more impactful. Second it also increased the learning curve for new players since this siystem provides a more clear overview. First the actual rank up per experience is streamlined as the bonus is. The accuracy bonus has been removed since it only provided artillery with that bonus and most numbers were confusing for starters.

Im sorry for the wall of text but this has been a project I have been taken since I have been active in the TD community. Most changes you guys have read so far (even in AoA´s PR) have been around since february I believe through Anjew, @CDVoidwalker and me but have been dropped due to AoA´s concerns.

There was even a suggestion PR on Github if I´m not mistaken but I can´t find it atm.

Anyway I will continue testing and will try to test out more features to imrpove the environtment of TD and I will also ask the people to maybe atleast have a look on this PR.

If there is anything else I can do than please @pchote contact me.

pchote commented:
> the current Light Support weakens Nod´s early game by a lot

How specifically so?  This could potentially be re-balanced to be fairer without completely gutting it.

ZxGanon commented:
> How specifically so? This could potentially be re-balanced to be fairer without completely gutting it.
That is specifically caused by the light vehicles in light support.

The Humvee is better than the Buggy. Balance wise the production of both units is pretty fair and Buggies can even trade cost efficient in even value numbers. But at the beginning the only thing Nod can do with the buggy is running away while the GDI can claim map control. Also GDI can pressure with their light support way better and probably even kill the Nod player at the beginning.

Also what I have noticed is that AoA´s change of APC projectile also affects the anti air weapon. Now APC´s tend to miss their air targets when flying by which can be a big issue in GDI mirror. I think that must be a mistake that he didnt split the ground and the anti air weapon.

pchote commented:
Would replacing some of the infantry with a second buggy unbalance the light support too much in the other direction?

ZxGanon commented:
> Would replacing some of the infantry with a second buggy unbalance the light support too much in the other direction?
Ufff well see: If the same ammount of money value (like 3200$) of Humvee and Buggy fights each other the Buggies will always come on top with around 2-3 Buggies being alive at the end.

It is kinda hard to try balancing that out with adding units since the Vehicles are so different on their own.

I would rather try leaving it for people that like it and adding a new one because thats saves lots of trouble.

AoAGeneral commented:
The listed changes are a collection of community feedbacks. Im not sure where you are getting these changes in specific are being just mine. I have stated this many times, you need to read the forum post labeled TD Balance Thread as the entire paged category is a long listed history of bouncing ideas back and forth between members. The comment about the changes not being tested is something that has been up for about two months.

Under that note:

Decreasing the price of the MRLS is a mistake due to their AoE damage and able to kill light vehicles, it is a step I would rather take slowly. There is already debate about this unit, and the Nod artillery to which will need collective debate in the future.

Why is the E3 movement speed to be increased? Doesn't this promote a major buff to infantry and base rushing tactics? The reason E3 move slower is prevent these kinds of all in moves and a plague of infantry dominating the map. (Samiliar to how RA1 works).

MCV - Please read the forums about this topic. This is explained in pages 1-10 and a main reason why it got an HP buff. This is a 3 year old issue that I do not want to see happen again without additional team game testing. Currently in 1v1 games with it being 3500$ it is an investment build. Bringing this down to 3000$ risks the issue of multiple MCVs being churned out and causing another base epidemic problem.

Changing the scout squad setup promotes barrack opener plays. Which again, turns into another restricted build order if players rush out engineers. Starting with a hummer/buggy prevents engineer rushes to grab the oil unless the players goes with another strategy. However, if this is to be added on and not change the original Light Support setup then that is fine.

Airstrikes coming from the north and south only makes these even worse. If units are closer to either the north or south, there is a 1 out of 2 chance that they will appear very quickly. Keeping them to come from all sides allows a 1 our of 4 chance that they will appear off screen quickly. North and South directions also creates a problem of angled bombing. There are some structures and maps where airstrikes strafing from left to right is much better then just north or south.

The new veterancy would cause issues for aircraft units due to their price. (All be it any expensive unit would suffer from this while cheaper units would gain benefits more quickly.) Does this buff a large majority of Nod units due to their price?

ZxGanon commented:
> The listed changes are a collection of community feedbacks. Im not sure where you are getting these changes in specific are being just mine. I have stated this many times, you need to read the forum post labeled TD Balance Thread as the entire paged category is a long listed history of bouncing ideas back and forth between members. The comment about the changes not being tested is something that has been up for about two months.
I gathered my feedback from the TD community aswell and they seem to feel listened more to me.

> Decreasing the price of the MRLS is a mistake due to their AoE damage and able to kill light vehicles, it is a step I would rather take slowly. There is already debate about this unit, and the Nod artillery to which will need collective debate in the future.
The Nod artillery straight outperforms the MLRS in almost all cases except damage against Light and its Projectile speed. In theory the prices should even be swapped if the artillery of Nod wasnt so clunky to use.

> Why is the E3 movement speed to be increased? Doesn't this promote a major buff to infantry and base rushing tactics? The reason E3 move slower is prevent these kinds of all in moves and a plague of infantry dominating the map. (Samiliar to how RA1 works).
Again a request by the community since the Rocket Soldier is always lacking behind especially when you lay Nod. Also haven seen any base rushes yet. Only under recent TD openra where the APC is broken and literally everything works with it.

> MCV - Please read the forums about this topic. This is explained in pages 1-10 and a main reason why it got an HP buff. This is a 3 year old issue that I do not want to see happen again without additional team game testing. Currently in 1v1 games with it being 3500$ it is an investment build. Bringing this down to 3000$ risks the issue of multiple MCVs being churned out and causing another base epidemic problem.
Your issues with the MCV are all outdated and also were caused by the old mappool to cause even more basepush issues and mcv spamming with 2000$ MCV. 3000 MCV is nowhere near that power level and even Anjew confirms that and he was the leader of hating 2000$ MCV. I have only seen people losing so far if they tried going for a second MCV early on. It takes quite the patients and passiv play to make it work. The MCV HP reduction is caused by the massiv MCV monster trucking allins you can perform but you havent witness it like the APC since 4 month waiting time.

> Changing the scout squad setup promotes barrack opener plays. Which again, turns into another restricted build order if players rush out engineers. Starting with a hummer/buggy prevents engineer rushes to grab the oil unless the players goes with another strategy. However, if this is to be added on and not change the original Light Support setup then that is fine.
It is gonna be an extra and not change the Light Support if you would have play Mesacers and my maps you would have known. On a site note people still avoid using the Engineer because it sadly doesnt pay off atm. Gonna have to find a fix for that later.

> Airstrikes coming from the north and south only makes these even worse. If units are closer to either the north or south, there is a 1 out of 2 chance that they will appear very quickly. Keeping them to come from all sides allows a 1 our of 4 chance that they will appear off screen quickly. North and South directions also creates a problem of angled bombing. There are some structures and maps where airstrikes strafing from left to right is much better then just north or south.
So far the community has been very happy with that change since it is less RG when you whole army gets eradicator by an one click ability. Now it is very strategic to engage on nothern or southern border lines and also kind of mind gaming to set it in the middle of the map. Also even if the AS was centered into the middle of the map it still does kill armies because sometimes you cannot run away from the position you are in atm.

> The new veterancy would cause issues for aircraft units due to their price. (All be it any expensive unit would suffer from this while cheaper units would gain benefits more quickly.) Does this buff a large majority of Nod units due to their price?
Excellent question. Yes cheaper units can certainly hit Heroic quicker than other more expensive units if you compare Minigunners or Tanks but this has been the case with the old Veterancy system aswell and the old is even worse due to the fact that you can reach the first 2 level quicker than on our new system. Also the more expensive units tend to either be more tanky like Mammoth Tanks or stealthed and hard to catch like the Stealth Tank. Anyway there is gonna be research on the Veterancy System since even though it works lovely for Shattered Paradise that it doesnt have to work for TD. But that needs testing and also in the end @pchote needs to decide if he used the old one, my new sytsme or his own.

pchote commented:
Re veterancy: could you create a new issue specifically for discussing that?  That is going to need to be a longer term project with a lot of research and testing, so this PR isn't a good place for it.

ZxGanon commented:
> Re veterancy: could you create a new issue specifically for discussing that? That is going to need to be a longer term project with a lot of research and testing, so this PR isn't a good place for it.
Will do so.

AoAGeneral commented:
It doesn't matter who listens to who more. They gathered feedback. Period. If someone suggests "Hey can we have more power plant HP?" Then im gonna look into it and add it when possible. Do not disregard.

"The Nod artillery straight outperforms the MLRS in almost all cases except damage against Light and its Projectile speed. In theory the prices should even be swapped if the artillery of Nod wasnt so clunky to use."
---Which is what makes the unit vastly different. This doesn't make sense to me since the artillery projectile is a slower travel speed and faster units can escape it while the MRLS is much faster and more destructive to faster units. Not agreed with this here and best left for future testing.

The E3 speed change turned the games into a RA style game where infantry become dominant and destroying vehicles rather easily. Increasing the E3 speed also decreases the amount of time any infantry needs to wait for E3 which causes a balance shifting problem.

The problems are not outdated. If a problem is fixed, and then you make that change again to what it used to be, you re-invent the problem. If I change bikes to a price of 600, I just re-invented the problem. If maps are the cause for MCV spam problems, then I suggest keeping it at 3500$ until further testing is done on those old maps to see what happens. I have explained before, I am extremely paranoid of MCV being bit to easily and making MCVs cheaper and weaker makes them easily killed. Decreasing their build time is also a large mistake at 47 seconds because you are now double promoting the issue. Reducing its speed is a large mistake because now Mammoth Tanks are able to keep up with them longer.

Don't need to play maps to understand if Scout Squad is an addition or replacement. You can easily explain that here.

As mentioned before, Airstrikes kill certain buildings more effectively strafing from the left and right. When HP armor types get added in this will effect Airstrikes greatly and allow them to be balanced out more vs damage type. I much prefer a directional choice for Airstrikes then a more narrowed chance that they can spawn from the edge of the screen. At least with the original way there are three other directions they can come from creating less chance.

ZxGanon commented:
> Which is what makes the unit vastly different. This doesn't make sense to me since the artillery projectile is a slower travel speed and faster units can escape it while the MRLS is much faster and more destructive to faster units. Not agreed with this here and best left for future testing.

So that means I only build Tanks and infantry against MLRS from now on following your logic and my opponent has wasted over 900$ building a unit that does almost nothing against my army.

> The E3 speed change turned the games into a RA style game where infantry become dominant and destroying vehicles rather easily. Increasing the E3 speed also decreases the amount of time any infantry needs to wait for E3 which causes a balance shifting problem.

I dont know why you are always so damn afraid of RA it doesnt really make sense to me. What is the problem with having stronger infantry pushes when Rocket Solders can travel faster. The realism is also kinda kept since they still move slover than Minigunners. Also infantry can be very easily counter be either spamming Humvees/Buggies or setting up artillery (well the MLRS is only good against light armor but you can also use grenadiers).

> The problems are not outdated. If a problem is fixed, and then you make that change again to what it used to be, you re-invent the problem. If I change bikes to a price of 600, I just re-invented the problem. If maps are the cause for MCV spam problems, then I suggest keeping it at 3500$ until further testing is done on those old maps to see what happens. I have explained before, I am extremely paranoid of MCV being bit to easily and making MCVs cheaper and weaker makes them easily killed. Decreasing their build time is also a large mistake at 47 seconds because you are now double promoting the issue. Reducing its speed is a large mistake because now Mammoth Tanks are able to keep up with them longer.

The 47 seconds of build time can also be changed to 50 seconds I just had to check and wanted to have it close to 48 seconds somehow. But even after 5 days of testing (even in teamgames) people even try to pull of quick (even midgame) multiple MCV´s but the game is either over by than or they die themselves because the opponent noticed.

> As mentioned before, Airstrikes kill certain buildings more effectively strafing from the left and right. When HP armor types get added in this will effect Airstrikes greatly and allow them to be balanced out more vs damage type. I much prefer a directional choice for Airstrikes then a more narrowed chance that they can spawn from the edge of the screen. At least with the original way there are three other directions they can come from creating less chance.

Yes that one is a solid argument. Well like I said I wanted to finally adjust the stupid RNG that exists in TD by changing the spawning direction of AS, But I already talked with Mesacer and he is willing to code a new way where you can drag the AS into the position you want it coming from. That would be the ultimate solution.

AoAGeneral commented:
Are you saying this in a sense of Tanks + Infantry vs MRLS only?

ZxGanon commented:
> Are you saying this in a sense of Tanks + Infantry vs MRLS only?
Nothing goes in TD without combining units but the Nod Artillery is good against everything literally. And even though tanks are fast they either get stopped and shred by them or drive through my army and get killed by my tanks.

The MLRS cost reduction was to give this weak unit atleast finally a space of production since it was too expensive to effort (you just build more medium tanks instead).

AoAGeneral commented:
Because when the unit (MRLS) is combined in a force they do a lot of damage. Their main strength is their accuracy and AoE spread damage vs units.

ZxGanon commented:
> Because when the unit (MRLS) is combined in a force they do a lot of damage. Their main strength is their accuracy and AoE spread damage vs units.

Well everything is strong and does tons of damage when combined so thats...
Yeah thats its strength thats why I made it 800$ and not 600$ like the Nod Artillery. The MLRS is ust not worth 1000$ nor even 900$. One of the many reasons GDI late game is weaker compared to that of Nod (well except the broken Ion Cannon that one shots your army without any warning nor abilitiy to dodge it). Well atleast Ion Cannon is not random like AS.

gdavegdave commented:
I'm encouraged to see a community-backed nerf to the APC in this PR despite your reservations, AoA. Continued testing on the available mod maps should indicate whether the values work. As I said to you in private, I think there have been some communication issues between yourself and the community in the past and I'm encouraged by our chat that things will be better in future. 

I'm glad to see that Mesacer/teinarss, with his coding skills, is working with you, AoA, and that he has already submitted a PR on the repair pad: https://github.com/OpenRA/OpenRA/pull/15163

ZxGanon commented:
This is a map for testing balancing changes in TD:

General:
New starting units:
* Scout Squad which provides you with 3 Minigunners
Airstrike:
* Changed so that it only appears from either north or south, still random appearance
* Changed spread of the machine gun from 426 to 386
Low Power Slowdown:
* To fight being on low power the build time of the Power Plant has been reduced from 12 to 8 seconds
MCV:
* Cost is reduced to 3000$
* Build time changed from 60 to 47 seconds
* HP reduced from 120000 to 60000
* Speed reduced to 60
Service Depot:
* Heal rate increased by 100%
* Decreased Power Requirement from -30 to -20
Oil Derrick:
* Reduced the health from 100000 to 70000
Rocket Solider:
* Increased movment speed from 42 to 50
Engineer:
* Fix for handling the problem when caputuring a building which gets repaired
Other:
* Husk from light vehicles on stays for a few secs
Weapons Factory / Airstrip:
* Reduced the power requirements from -50 to -40

GDI:
APC:
* Reduced the projectile speed from instant to 900
* Price increased to 600$
* Health reduced from 21500 to 19000
* Reduced the turn speed from 8 to 5
* Build time set to 15 sec
* Anti air gun reduced from 7 to 5
* Visionradius reduced from 7c0 to 6c0
Rocket Launcher:
* Cost reduced to 800$
Commando:
* Reduced the price from 2000$ to 1200$
* Reduced attackrange from 8co to 7c0

Nod:
Mobile SAM:
* Cost reduced from 600$ to 500$
SAM Site:
* Cost reduced from 650$ to 550$
* Build time reduced from 27 to 25 sec
Apache Longbow:
* Increased attackrange against ground from 4c0 to 5c0 to be on par with Orca
* Increased damage against None from 100% to 125%

Those are the changes currently being tested. 
Future adds:
* Reducing the cost and build time of SAM Site even further
* Splitting anti ground and anti air weapon of APC and reintroducing instand hit projectile of the AA weapon

Still have a closer look on the commando and MCV HP but it still looks good so far.
The airstrike change still imrpoved the gameplay in terms of AoE but if a new code is gonna be introduced I have no hard feelings dropping it. Right now I just wanna see a change to AS.

teinarss commented:
I'm starting to get really tired of this dumb fighting, there is no backing from the community  for this. The community is not trying to do a revolt.  It was frustrated on the slow response for changes on the APC.

So can we keep this friendly and continue to make this game better!

The games I have played and also seen a lot of replays played on AoAGeneral maps these games have been great, no problems with APC spam.  I think these changes are enough for this release. And I also thinks that it's better to be a bit sparse with the changes than to change the whole game in one release. 

**What we get is:**

A fixed APC which was needed. 

A change on the power which changes the flow of the game in a good way, this was suggested by GDave. 

We also get a lower cost on the MCV which is long overdue. 

A fix which soften the blocking mechanics in early game play.

**The future** 
So moving forward from this, GDave, AoAGeneral and I had a talk about this and came to the conclusion that I will help with the balancing, mostly with the technical part but also with communication with the community. 

**Transparency**
We have to come to conclusions that we need to be a bit more transparent and communicate our intentions with the community.

**Involvement** 
One positive thing I want to keep from this is the involvement from the community. I would even like to see even more involvement from the community. It would be cool if people posted stuff like this:

“I think x should be changed because of y, look at this replay. What are your thoughts?” 

Or

“I don't agree with these changes because of x, look at this replay“

ZxGanon commented:
There is still a huge issue with AoA´s PR. The APC does miss against moving Air and that makes the APC amt useless in GDI mirror so you gotta use Orcas to defend against Orcas.

There are 2 ways of doing this:
* Either keeping the slow projectile against air but increasing it´s attack range against Air
* Or making the projectile of the AA wepaon as fast as it used to be but not touching the range

Both changes would be fine even though I tend to go for the 2nd idea of changing the projectile speed against Air to make the Mobile Sam of Nod special and also making more sense since its using huge rockets.

Also AGT´s seem to be overperforming atm (well they were doing it before too) because you can eaisly que them without any hesitation. 1000$ over 48 seconds is basically nothing it does only little to drain your money which means you can constantly spamming it.

AoAGeneral commented:
The projectile speed only applies to ground units not air. APC gun has two weapons. APCGun and APCGun.AA. The APCGun.AA speed was never touched and remains the same speed.

AGTs are to be looked at later in the playtest. The AGT is currently being looked at and tested but it should not be fixed in this PR as it may present the same problem as #6825

ZxGanon commented:
> The projectile speed only applies to ground units not air. APC gun has two weapons. APCGun and APCGun.AA. The APCGun.AA speed was never touched and remains the same speed.

Are you really sure about this because on my mod map I used your values for APC and weapon. Anyway it´s an easy fix if there is something wrong so nothing to be scared of.

Oh yes another request would be to nerf the AS further. The spread change didnt really do much. I thought about reducing the damage of the Machine Gun against Light armor so that it only obliterates infantry but only the napalm kills light and heavy vehicles.

reaperrr commented:
@ZxGanon 
https://github.com/OpenRA/OpenRA/pull/15150/files#diff-5930973a0d3e1fd42e2a8991e3ccbf26L155
So the AA projectile speed didn't change.

@AoAGeneral Needs a rebase.

gdavegdave commented:
I wonder if the AA range is a little short now. It went from 7 to 5; perhaps 6 would work better.
 
I haven't played many games on the balance test-maps, less so when an enemy went air. When they did go air, it felt pretty hard to hunt down helis with APCs.

ABrandau commented:
Also, I recall 5A complaining that engineers dont have selection priority on TD, you guys might want to check that.

ZxGanon commented:
> Also, I recall 5A complaining that engineers dont have selection priority on TD, you guys might want to check that.

Yeah that´s true they are selected with army which is very annoying.

AoAGeneral commented:
@reaperrr fixed! ^^

@gdavegdave added and adjusted.

@ABrandau Added Engineer selectable priority.

@ZxGanon The AS has already been nerfed to the ground. I would prefer to have this implemented #14991 which will allow damage adjustments from area of effect type weapons to be adjusted (Such as nukes, ions, airstrikes,)

AoAGeneral committed:
Update TD balancing:

Apache damage vs infantry at prone increased from 50 percent to 80 percent.

  Apache damage was cut in half making them less effective versus infantry. Giving them an extra boost damage wise versus prone is enough then going with a universal damage boost.

MCV Price from 4000 to 3500.

  Some say they want 3000 but im still paranoid of the epidemics that have happened in the past with 1v1 and team games. Needs more team game tests if they want 3000.

Repair Pad decrease power from -30 to -20.

 Power consumption is a little high. Specifically if they want to build extras.

APC Price from 550 to 600.

APC HP from 21500 to 19000.

APC turn speed from 8 to 5.

APC build duration from 900 to 938.

APC AA range from 7 to 6.

APC reduce vision range from 7c0 to 6c0.

APC projectile speed reduced from 2c0 to 900.

  See notes below.

Power Plant buff HP from 50,000 to 55,000

  Makes power plant sniping a little tougher. But still effective to take out.

Change power structures (Airstrip/Factory and Refinery to 40 power)

  Important change. This allows more diverse builds to happen such as now able to build refinery > barracks > factory. This combo helps easy defends with quick vehicle herasses.

MRLS Price reduction from 1000 to 900.

  Allows an easier method to build on the field. A unit that is often times killed by airstrikes allows more forgiveness when on the field.

Light vehicle husks HP reduce to 2000. (reduce timer on field wreckage.)

  Prevents early on refinery blocks and other pathing unit issues.

Oil Derrick reduce HP from 100,000 to 80,000

  Oil Derricks were a little to strong. Easily can be killed now.

Change power plant build duration from 12 seconds to 8 seconds. (Low power build duration from 24 seconds to 16 seconds.) Note: Does not include Adv. Power plant

  Due to the struggle of low power this enables players to build regular power plants to help get back up quickly. This however, does not apply to Adv. power plants. The outcome is then scattered extra power plants rather then Adv. power due to the player being on low power.

Change engineer capture threshold from 50 to 55 and added a lower selection priority.

  Fixes the structures from being fixed and preventing the second engineer from getting canceled out or being wasted on damaging the structure again.

NOTES:

I have mentioned that I disagree on the above APC changes. The reasons I conclude may happen is buffing the bikes with a lack of counter. This may result in games where infantry are now used to help counter against buggy/bike play rather then an APC/infantry play or APC/hummer play. If and when players speak about bikes being OP please look back at the changes that happened in the past.

pchote commented:
I've squashed the commits and changed the first commit line to make this clearer in the commit history.

pchote commented on mods/cnc/rules/defaults.yaml:
```diff
@@ -957,6 +959,23 @@
+	Health:
+		HP: 2000
+
+BGGY.Husk:
```
These don't belong in defaults.yaml.  I will push a fix before merging.

pchote reviewed (approved):
While there may still be discussion on what *more* to do, the changes as they are here appear be uniformly wanted.  Lets take this, and then see what playtest feedback has to say.

Personally, I am not very happy with the inconsistencies introduced with the Light husks (APC is counted as "light", while the lighter armored rocket launcher and supply trucks are "heavy") but i'm willing to overlook that for now, and hope the situation can be improved in the future.

AoAGeneral commented:
Im willing to go into discussion about it. Let me know when you would like to do so ^^

pchote commented:
https://github.com/OpenRA/OpenRA/wiki/Changelog-(bleed)/_compare/f7562e0776fe6df302e94d0f0c825e798e87da69
